wiki/osu!wiki_contribution_guide/en.md

```diff
@@ -0,0 +1,724 @@
+# How to Contribute to the osu!wiki
+
+Thanks for your interest in making osu!wiki better!
+As you can see, everything has been moved to [GitHub](https://github.com/ppy/osu-wiki).
+
+So here's a few things you should have in mind to get you started for contributing to the osu!wiki.
+
+## Disclaimer
+
+This quick-start guide was intended for newcomers not familiar in the workflow of GitHub.
+If you have previous experience in using GitHub, you are free to skip this guide.
+Do note that there are other ways/shortcuts in solving the problem; this guide is simply one of the many ways of solving the problem.
+
+When starting out, it is best to start on GitHub's way first, and slowly learn the terminal Git way properly in the meantime.
```

Comment:
Learn the GitHub workflow first, by using the features provided by GitHub.

Use terminal Git if you plan to go serious.

^ Something like that ^